### ISSUE TITLE
Manuscrito

### ISSUE THREAD

**Issue by BidelkisCastillo** [biogeografia-201902/datos-hormigas-campus-uasd#2]
Buenas tali, me esta dando problemas subir los datos al Chunk de R, como lo hago.  @geofis

**Comment by geofis** [comment]
Dime qué problemas específicos.

**Comment by BidelkisCastillo** [comment]
cuando estoy subiendo los datos de las hormigas en el chunk de rstudio (servidor) me dice error in file rt, he buscando en to los laos, pero para mi parecer no se puede hacer eso en el servidor

**Comment by BidelkisCastillo** [comment]
aunque yo comence a hacerlo en la rstudio desktop

**Comment by geofis** [comment]
El error `rt` se refiere normalmente a que no encuentra un archivo. De todas formas, quizá no estoy entendiendo bien. ¿Te refieres a que tus datos no los puedes subir al servidor? Si quieres seguir trabajando en RStudio Desktop, y puedes tejer y sincronizar con GitHub, pues sigue en RStudio Desktop, pero haz commit>push al final. Ten en cuenta que no es recomendable realizar actualizaciones al repo de GitHub desde varias fuentes; no es que no sea posible, pero se complica.

**Comment by BidelkisCastillo** [comment]
ok, lo voy hacer en R Desktop, cualquier duda le seguire escribiendo

**Comment by BidelkisCastillo** [comment]
pero no es por la ubicacion del archivo, porque la subi en rstudia normalito

**Comment by BidelkisCastillo** [comment]
profesor no encuentro la funcion para hacer las kables, auxilio, recuerde que estoy en rstudio

**Comment by BidelkisCastillo** [comment]
creo que el truco esta aqui source('src/funciones_analisis.R'), pero como lo consigo y lo llevo a rstudio? S.O.S

**Comment by BidelkisCastillo** [comment]
YA ENCONTRE EL SRC, AHORA COMO LO LLEVO A RSTUDIO SIN HACER TANTO RUIDO, O SEA, SIN LLENAR UN CHUNK COMPLETO

**Comment by BidelkisCastillo** [comment]
taliban, dejelo, usted no responde temprano, no se para que sirve no

**Comment by BidelkisCastillo** [comment]
ya lo encontre

**Comment by geofis** [comment]
Je je, resolvió ella solita. Menos mal. Estoy en línea de nuevo

**Comment by BidelkisCastillo** [comment]
tengo otro problema, no encuentro gpkg de la funcion map, auxilio, me desmayo jajajaj

**Comment by BidelkisCastillo** [comment]
me duelen los ojos

**Comment by BidelkisCastillo** [comment]
porque r da tantos problemas?

**Comment by geofis** [comment]
¿Puedes al menos poner en pantalla el error, al menos con una captura de pantalla? Si te refieres al archivo geopackage que utiliza la función map, éste se encuentra en la ruta del repo de análisis de datos. Pero dame más datos para poder ayudarte.

**Comment by BidelkisCastillo** [comment]
la encontre y lo intente modificar pero me dio otro error

**Comment by BidelkisCastillo** [comment]
tmap mode set to interactive viewing
Error: Cannot open "C:\Users\william roa\Desktop\parcelas_tipo.gpkg"; The source could be corrupt or not supported. See `st_drivers()` for a list of supported formats.

**Comment by BidelkisCastillo** [comment]
como que el archivo esta corrompido, como el pld

**Comment by geofis** [comment]
No me muestras la línea de código que te da ese error. en todo caso, la función `mapa` asume que tanto el geopackage, como otros archivos, se encuentran en una ruta concreta. ¿Podrías mostrar el código que te da este error? Imagino que el repo `datos-hormigas-campus-uasd` lo tienes clonado en tu PC, en alguna ruta con una carpeta sólo para él. Al lerr el mensaje `C:\Users\william roa\Desktop\parcelas_tipo.gpkg` , me temo que no tienes el repo clonado en una carpeta.

**Comment by BidelkisCastillo** [comment]
no, yo lo que hice fue crear un chunk solo para eso, como lo logro conectar?

**Comment by BidelkisCastillo** [comment]
mapa <- function(vari, filtusuario = NULL, fun = mean) {
  #Genera mapas de variables/usuario
  require(sf)
  require(tmap)
  require(tidyverse)
  tmap_mode('view')
  parcelas <- st_read("C:\\Users\\william roa\\Desktop\\parcelas_tipo.gpkg", quiet = T)
  todos_los_habitat <- read.csv("C:\\Users\\william roa\\Downloads\\convertcsv.csv")
  todos_los_nidos <- read.csv("C:\\Users\\william roa\\Downloads\\todos_los_nidos.csv")
  todos_los_habitat <- colus(todos_los_habitat)
  todos_los_nidos <- colus(todos_los_nidos)
  todos_los_habitat <- usuarioanombre(todos_los_habitat)
  todos_los_nidos <- usuarioanombre(todos_los_nidos)
  tipomuestra <- ifelse(any(grepl(filtusuario, c('dahianagb07', 'enrique193', 'maritzafg'))),
                        'nidos', 'habitat')
  if(tipomuestra == 'habitat')
    {parcelas <- parcelas %>%
      inner_join(todos_los_habitat %>% dplyr::select(-tipo), by = 'parcela')}
  else if(is.numeric(todos_los_nidos[,vari]))
    {parcelas <- parcelas %>%
      inner_join(todos_los_nidos %>% dplyr::select(-tipo), by = 'parcela') %>%
      group_by(usuario, parcela) %>% summarise_if(is.numeric, fun)} else
        {parcelas <- parcelas %>%
          inner_join(todos_los_nidos %>% dplyr::select(-tipo), by = 'parcela') %>%
          group_by(usuario, parcela) %>% summarise_if(is.factor, paste, collapse = ',')}

**Comment by BidelkisCastillo** [comment]
le agradeceria que me dijera como hago esa conexion del repo, recuerde que estoy rstudio desktop

**Comment by geofis** [comment]
No se puede sólo creando un chunk. Toma nota:

1. Abre una instancia de RStudio (si tienes una abierta, puedes abrir otra).

2. Clona el repo de datos hormigas. IMPORTANTE: en una carpeta localizada FUERA de la de tu repo de manuscrito.

3. Has todos los mapas que quieras hacer.

4. Captura pantalla y cópiala en la carpeta de tu repo de manuscrito.

5. Insértala en tu archivo `manuscrito.Rmd` mediante `![Pie de figura](nombredearchivo.jpg)`

**Comment by BidelkisCastillo** [comment]
ok, y como le consigo el scritreproducible?

**Comment by geofis** [comment]
Asumiré que has creado un script para ello, no te pediré que incluyas dicho chunk dentro de tu repo. Una opción sería que, si quieres poner el bloque de código que usaste para generar el mapa (para fines no sólo de reproducibilidad, sino de historia), puedes crear un chunk con la etiqueta `eval=F` antes de la imagen, algo tal que:

    ```{r, eval=F}
    CODIGO QUE PRODUJO EL MAPA
    ```

**Comment by BidelkisCastillo** [comment]
buenas tardes tali, cuando estoy haciendo las matrices me da un error que dice que no encontró el objeto nomlat, voy a los datos y hago un str y veo que dicha variable no está presente, de donde rayos usted consigió esa variable, gaste las pocas neuronas que me quedaban ayer, responda sil vous pleit

**Comment by BidelkisCastillo** [comment]
estoy trabajando con la funcion matriz_comunidad_hab

**Comment by BidelkisCastillo** [comment]
la acabo de buscar en pooled habi y no veo esa variable

**Comment by geofis** [comment]
Puedo ayudarte más si colocas el código que produce dicho error aquí en el issue

**Comment by BidelkisCastillo** [comment]
ya encontré el problema , es que no había cargado la base jijijijijin , mala mia. 😃😃😃😃😃😃

**Comment by BidelkisCastillo** [comment]
pero este pendiente que voy a seguir jodiendo, quiero salir de esto jejejejejje 🤪🤪🤪🤪👩‍🔬👩‍🔬👩‍🔬👩‍🔬

**Comment by geofis** [comment]
je je, autocapacidad resolutiva, mejor

**Comment by BidelkisCastillo** [comment]
Tali, estamos en la face final, esto es de Arabes, por favor denos un plazo hasta mañana para entregar, mañana si entregamos bien.

por amor a las hormigas



Bidelkis Castillo

Analista Geomática del Sistema Integrado Nacional de Información (SINI)

Comisión Nacional de Emergencias (CNE)

bidelkiscs@gmail.com

Insistir, persistir y nuca desistir. Lo mejor esta por llegar.

________________________________
De: José Ramón Martínez Batlle <notifications@github.com>
Enviado: miércoles, 18 de diciembre de 2019 10:43 p. m.
Para: biogeografia-201902/datos-hormigas-campus-uasd <datos-hormigas-campus-uasd@noreply.github.com>
Cc: BidelkisCastillo <bidelkiscastillo@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [biogeografia-201902/datos-hormigas-campus-uasd] Manuscrito (#2)


je je, autocapacidad resolutiva, mejor

**Comment by geofis** [comment]
Pues pa' mañana entonces. Comunícalo a las demás personas.

**Comment by BidelkisCastillo** [comment]
Tali como puedo puedo subir los datos SRC, porque ya tenia clonado el repo cuando usted subió eso.

**Comment by BidelkisCastillo** [comment]
Hey, tali, le envié el RMD, porque trabaje local en mi PC, lo subí directo a Github

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by geofis** [comment]
Imagino que tuviste problemas para tejer. Intentaré tejerlo yo y te aviso si necesito algo.

**Comment by BidelkisCastillo** [comment]
De acuerdo

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by geofis** [comment]
Vi el RMD que está en el servidor, el cual me dices haber subido desde tu PC, pero no están escritas las secciones Resultados y Discusión, ni hay script reproducible. ¿Estás segura de que el RMD que está en el servidor es correcto?

**Comment by BidelkisCastillo** [comment]
El del servidor es el viejo, no pude trabajar ahí, y lo hice local, y subí el RMD nuevo a Github directo

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by geofis** [comment]
OK, lo subiste a GitHub. Miro por allá. De todas formas, comprime la carpeta que tienes en tu PC con el repo del manuscrito, y envíala por correo entonces, a ver si puedo tejer el RMD.

**Comment by BidelkisCastillo** [comment]
Y no puede subir el RMD A R Y VERLO, porque dejé mi PC anoche dónde mi abuela, tendria que ir a buscarla y estoy lejos ahora. Deje ver cómo lo hago.

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by geofis** [comment]
Sí, puedo y de hecho pude, pero seguramente hay archivos asociados que sólo están en la carpeta de la PC donde trabajaste.

**Comment by BidelkisCastillo** [comment]
Todo con lo que trabaje lo descargue del repo

Obtener Outlook para Android<https://aka.ms/ghei36>

________________________________

**Comment by geofis** [comment]
OK, voy a intentar el tejido y si pide archivos que no estén en los repos, y que sean tuyos, pues te aviso.
